src/data/analysis/Strategy Helpers/Faxes and Summons.mdx

```diff
@@ -0,0 +1,102 @@
+| modern zombies | The Defiled Alcove | Modern zombies are guaranteed at +850% initiative, which is (startlingly) pretty straightforward to get in most paths nowadays. But why not summon one up and avoid the hassle entirely? As moderns clear 5 evil apiece, you can oust the Alcove's 37 evil with **eight copies** of the zombies. |
+| dirty old lihc | The Defiled Niche | The niche is one of the longest zones; with no speedups (as we have right now in standard), it takes **twelve whole dirty ol' lihcs** to clear the zone. This makes it slightly less compelling than the other Cyrpt speedups as a summon, because you need to devote a whole host of copiers if you want to solve the Niche by copying DOLs. But it IS possible; two habitats and two mimic eggs does the trick. Still, it's a little below the other cyrpt speedups, as it exists in a 4-monster zone where a few banishes and a macro can generate it anyway, and you need to fight so many of them that it does start to become a big resource drain to do the Niche via summons and copiers. |
+| Baa'baa'bu'ran | The Hidden Temple | This is a great summon; it's rare that an ascender will -not- want to summon Baa'baa, as 3 stone wool allows you to do silly effect extension with Shadow Affinity while also getting you the requisite 4-turn unlock of the Hidden City you need to accomplish in any run. The only reason I put this in Tier 1 instead of Tier 0 is that you can replicate it with a clover, and in modern standard we have more clovers than we know what to do with, so you probably would prefer to use the clover. Still -- if you're in a no-clover path and lack access to the Deck of Every Card, Baa'baa is almost certainly on the menu, one way or another. |
+| The "tool" gremlins from the Yossarian war quest | The Junkyard | Usually, you spend about 2-3 banishes and a few macros on the Yossarian quest. You can avoid this entirely by simply summoning the tool gremlins, once you've acquired the molybdenum magnet. You need **one of all four**; that's a big gob of summons, but in our summon-heavy meta that's a tiny bit light on free banishes, this may actually become the meta answer for the Yossarian quest. |
+| pygmy accountant | The Hidden Apartment & The Hidden Office Building | Unlike butlers, accountants will give you the pages regardless, which makes them a nice summon that gets you quest progress; you need to fight **five accountants** per run to get your binder clip, and it doesn't particularly matter where you fight them, so it's pretty much fine to just throw them in a delay zone. The main reason you may not want to copy them: with 10 delay turns in the Hidden Office and 9 in the Hidden Apartment, you probably can encounter them naturally. But, the counter: in a NC-force heavy meta, there's a pretty strong case for just double forcing the Hidden Office NC and single forcing the Apartment NC, making the prospect of summoning and copying accountants into delay zones significantly more palatable. |
```

Comment:
Should probably put this after red butlers since it starts off with "Unlike butlers"